internal/pkg/partition/partition.go

```diff
@@ -90,6 +90,13 @@ func NewPartitionOptions(label string, uki bool) Options {
 			PartitionType:  LinuxFilesystemData,
 			Size:           0,
 		}
+	case constants.ImageCachePartitionLabel:
+		return Options{
+			FormatOptions:  *formatOptions,
+			PartitionLabel: label,
+			PartitionType:  LinuxFilesystemData,
+			Size:           0,
```

Comment:
should this be set to the size of the incoming cache?

Reply:
good question... I guess this makes sense (or specified as a parameter?)

Reply:
actually it shouldn't matter much here, probably the overall profile disk size should be bumped, another option is to increment the default with the size of cache , so it'll grow automatically

Reply:
ahh the image already supports setting disk size, we can document to bump it if using huge image cache